### ISSUE TITLE
Format alacchiReUnsupervisedState 

### ISSUE THREAD

**Issue by koustuvsinha** [ReScience/NeurIPS-2019#2]
Complete formatting of existing Latex files to [ReScience Template](https://github.com/ReScience/template).

**Comment by koustuvsinha** [comment]
CC-ing @gabrielalacchi, @carlito387 and @guillaumelam to check if all meta-data and organization is good. For any changes, submit a PR linking this issue.

**Comment by carlito387** [comment]
Hello Koustuv,
We modified the latex code of our article with the ReScience C template directly on Overleaf because we were not able to install and use properly the $make command.
Joint is the source code including the modified metadata.yaml and content.t​ex.
[alacchiReUnsupervisedState.zip](https://github.com/ReScience/NeurIPS-2019/files/4294649/alacchiReUnsupervisedState.zip)

We are not sure if the modifications we made on the metadata.yaml are good since we don't know what it will generate in the metadata.tex.
Basically what we want to fix from the metadata.yaml is the commas problem in the name of the authors (see picture), and we would like to add the † Equal contributions tag.

![commas_issue](https://user-images.githubusercontent.com/36552657/76016469-deb54000-5eea-11ea-93ac-271fd9e3e370.PNG)


Can you help us with that please ? 
Once we fix this problem, we will do the pull request.

**Comment by rougier** [comment]
You can generate the `metadata.tex` from `metadata.yaml` using the script in https://github.com/ReScience/template (look at the make script to see how to use it). Then you can upload the generated metadata.tex on overleaf.

**Comment by carlito387** [comment]
I managed to get it working. Thank you.
However the 'commas' problem I describe above is still present, and it seems that we can't use the usual dagger sign (†) as an affiliation for 'Equal contributions' in the metadata.yaml. When we do that, it is not properly reported in the generated article.pdf.

**Comment by rougier** [comment]
Do you have a pointer to your github repo for the paper (do you have one?) such that I can have a look?

**Comment by carlito387** [comment]
Here it is: https://github.com/carlito387/NeurIPS-2019

**Comment by rougier** [comment]
Using the latest [yaml-to-latex.py](https://github.com/ReScience/template/blob/master/yaml-to-latex.py) it seems to be working for me (based on your metadata file). Can you checkout the latest [template](https://github.com/ReScience/template) and re-try ?

**Comment by koustuvsinha** [comment]
@carlito387 You can resolve this using `$\dagger$` in your metadata.yaml instead of the unicode symbol. Tested on my end and it works.

**Comment by carlito387** [comment]
Thank you! I also have issues putting some text in bold. /textbf does not work unless I change the font formats which are set for the ReScience template... I can play with the text colors, but that's not a really nice solution... Do you have an idea about how this can be nicely done ?

**Comment by koustuvsinha** [comment]
@carlito387 can you update your PR and point me to the specific line?

**Comment by carlito387** [comment]
Hello, I would need some help on those remaining 3 issues we have (code available [here](https://github.com/carlito387/NeurIPS-2019/tree/master/alacchiReUnsupervisedState/journal)):

1- No ',' or 'and' separating the names of the second and third author. This issue is also present in the other papers.
![image](https://user-images.githubusercontent.com/36552657/76860812-0208b500-6832-11ea-8fec-0a50ff5c0ca5.png)


2- \subsubsection (lines 41 and 66 of content.tex):
In our original article: ![image](https://user-images.githubusercontent.com/36552657/76858510-f87d4e00-682d-11ea-9a75-0785d6ebc954.png)
Now: 
![image](https://user-images.githubusercontent.com/36552657/76858558-0af78780-682e-11ea-9871-fefc5255486a.png)

3- \textbf not working. I think liljeforsReUnsupervisedScalable also have this problem. The bold version of Source Serif Pro seems to be defined from line 923 to 928 of article.log, but I still have 3 latex font warnings, ont of them being:

> LaTeX Font Warning: Font shape `T1/SourceSerifPro(0)/bx/n' undefined
(Font)              using `T1/SourceSerifPro(0)/m/n' instead on input line 1.

Thank you for your help !

**Comment by rougier** [comment]
1. Can you post the corresponding latex code ?
2. It is a design choice (subsubsection = paragraph)
3. Not quite sure about this one. Probably the declaration of fonts miss the bold one in the template.

**Comment by carlito387** [comment]
1. This problem comes from the [metadata.tex](alacchiReUnsupervisedState/journal/metadata.tex) or the [metadata.yaml](alacchiReUnsupervisedState/journal/metadata.yaml).
2. Ok, thank you!
3. I'll continue to look at this problem...

**Comment by rougier** [comment]
Cannot access the link (private repo ?). But the point is that if you're familiar with latex, you can try to see what's wrong and why does the script fails at making it right.
